src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs

```diff
@@ -32,10 +32,12 @@ public sealed class PowerFxConfig
             DebugName = "DefaultConfig"
         };
 
+        internal readonly List<Action<IBasicServiceProvider>> AddFunctionImplementations = new List<Action<IBasicServiceProvider>>();

```

Comment:
> List<Action<IBasicServiceProvider>> [](http://example.com/codeflow?start=25&length=36)

?

Reply:
Used to "transfer" function implementations into the ServiceProvider (or any other better idea)